README.md

```diff
@@ -1 +1,114 @@
 # USWDS proposals
+
+> **Note**
+> At this time, we are accepting proposals only for new components. 
+>
+> You can submit requests for new features or enhancements by [creating an issue](https://github.com/uswds/uswds/issues/new/choose) in the uswds repo.
+
+This repo acts as a public hub to submit and discuss proposals for new U.S. Web Design System (USWDS) components.
+
+## Overview of the proposal process
+
+All requests for new USWDS components must be submitted as a proposal. Proposals are detailed documents that should make a compelling case for adding a new component to the Design System.
+
+Anyone can submit or contribute to a proposal. Community contribution is at the heart of this process and we encourage you to share your ideas, opinions, and votes.
+
+### Ways to contribute
+
+The proposal process exists to help the USWDS team better understand community interest in new component ideas. Your opinions matter, and we want to make it easy for you to contribute.
+
+To help guide you through the process, we provide the following resources:
+
+- [Instructions for submitting a new proposal](#submitting-a-new-proposal)
+- [Instructions for contributing to an existing proposal](#contributing-to-an-existing-proposal)
+- [Information about our evaluation process](#proposal-evaluation)
+- [Information about roles and responsibilities](#roles-and-responsibilities)
+- [A proposal template](https://github.com/amyleadem/uswds-proposals/blob/main/proposals/_proposal-template.md)
+
+
+#### Help with GitHub
+
+This process requires a knowledge of GitHub [branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) and [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests). If you need help submitting or contributing to a proposal, please reach out via [TK].
```

Comment:
Consider adding a link to this https://digital.gov/resources/an-introduction-github